src/main/java/net/darkhax/botanypots/block/BlockBotanyPot.java

```diff
@@ -331,16 +331,9 @@ public void onReplaced (BlockState state, World worldIn, BlockPos pos, BlockStat
             if (tileEntity instanceof TileEntityBotanyPot) {
                 
                 final TileEntityBotanyPot pot = (TileEntityBotanyPot) tileEntity;
-                
-                if (pot.getSoil() != null) {
-                    
-                    dropItem(pot.getSoilStack(), worldIn, pos);
-                }
-                
-                if (pot.getCrop() != null) {
-                    
-                    dropItem(pot.getCropStack(), worldIn, pos);
-                }
+
+                if (pot.getSoilStack() != ItemStack.EMPTY) dropItem(pot.getSoilStack(), worldIn, pos);
```

Comment:
Could you please change this check to `!pot.getSoilStack().isEmpty()` instead, and place `dropItem` within brackets like it was previously formatted?